hypothesis-python/src/hypothesis/strategies/_internal/types.py

```diff
@@ -195,6 +202,10 @@ def from_typing_type(thing):
             else:
                 literals.append(arg)
         return st.sampled_from(literals)
+    if is_annotated_type(thing):
+        args = thing.__args__
+        annotated_type = args[0]
```

Comment:
What if `args == ()`?  Should we check for this in `is_annotated_type()`?  If it's impossible, let's add `assert args, "it's impossible to make an annotated type with no args"` to make that obvious to later readers.